### DISCUSSION TITLE
Pró-labore de desenvolvedor

### DISCUSSION THREAD

**Discussion by caiquearaujo** [frontendbr/forum#2132] [Discussão]
Sou um desenvolvedor experiente, full-stack além de ser especialista em UX/UI. Eu tenho uma velocidade e agilidade de trabalho muito boa também, por isso não necessitei, por enquanto, construir uma equipe. Tenho fechado contratos muito grandes, chegando a 6 dígitos, recentemente. E, agora, apareceu a necessidade de formalização da PJ. 

Já compreendo a alíquota na classe 620, é possível chegar a 6% com manobras. A questão é: e o pró-labore? Eu, por exemplo, trabalho sozinho por enquanto. Consigo atender a demanda e a frequência de projetos. E os gastos mensais com equipamentos, softwares, internet, luz e afins não passam de R$ 3.000,00. Então, vamos supor que eu ganhe R$ 100.000,00 no mês. Só posso retirar 28% de pró-labore, ou posso retirar R$ 90.000,00, por exemplo?

Alguém já passou por isso?

**Comment by igorlvicente** [comment]
Até onde sei, você pode tirar até 100% de pró-labore, mas não vale a pena. Só vale a pena tirar 28% para se enquadrar no fator R. O resto você tira como lucro (e, portanto, livre de quaisquer impostos/taxas que caem sobre o pró-labore).

PS: Não sou especialista em contabilidade, mas acho que esse caminho vai te ajudar.

**Reply by danilovaz** [reply]
Exatamente isso. Tire do pró-labore o X que você deseja e distribuia o resto como lucros/dividendos que são isentos.

No fim do ano você declara que recebeu X de pró-labore pelo CNPJ da sua empresa como sócio e Y como lucros e dividendos da sociedade.

Bom ter um contador pra te auxiliar na prestação correta das informações. Vai ter contador que vai te falar pra deixar o pró-labore no mínimo e distribuir todo o resto como dividendos...aí você precisa analisar. Nesse caso você paga o mínimo de pró-labore e todo restante vem livre (fora os 6% que vc já pagou de imposto)

**Reply by lucas-barbosa** [reply]
Mas no caso essa distribuição de dividendos pode ser feita mensalmente ou apenas anualmente?

**Reply by danilovaz** [reply]
Pode ser feita mensalmente sim, na declaração vai declarar o quanto de dividendos recebeu durante o ano-calendário
